corehq/apps/cloudcare/static/cloudcare/js/formplayer/router.js

```diff
@@ -209,6 +209,9 @@ hqDefine("cloudcare/js/formplayer/router", function () {
             // force manual search in split screen case search for workflow compatibility
             sidebarEnabled ? { forceManualSearch: true } : {}
         );
+
+        var queryObject = _.extend(queryObject, {initiatedBy: initiator});
```

Comment:
don't need to initialize `queryObject` again just to use `_.extend`

Reply:
also I think you can just stick this in the initial definition after line 207

Reply:
Moved to initial declaration